quiz.js

```diff
@@ -17,6 +17,15 @@ db.dropDatabase();
 
 // your code begins here
 
+db.westeros.insert({houses:{name:"Arryn",
+motto:"As High as Honor"},
+{name:"Stark",
+motto:"Winter is Coming"},
+{name:"Targaryen",
+motto:"Fire and Blood"}
+});
+
```

Comment:
Unfortunately, this is syntactically incorrect. To insert multiple items all at once, you'd need to pass them in as an array, like this:
```
db.houses.insert(
  [
   {name : ''Arryn",
    motto : "As High as Honor"},
   {name : ''Stark",
    motto : "Winter is Coming"},
   {name : ''Targaryen",
    motto : "Fire and Blood"}
  ]
);
```